java11_debian/Dockerfile

```diff
@@ -27,7 +27,11 @@ RUN apt-get update && apt-get install -y docker-ce
 RUN wget https://dl.google.com/go/go1.15.2.linux-amd64.tar.gz
 RUN tar -C /usr/local -xvf go1.15.2.linux-amd64.tar.gz
 
-ENV PATH "/usr/local/go/bin:$PATH"
+RUN mkdir -p /go
```

Comment:
What do you think about using the same commands as the official go Dockerfile?
https://github.com/docker-library/golang/blob/071e264f53e89ea75f1a38f6c1c33641685d8560/1.15/buster/Dockerfile#L119-L122https://github.com/docker-library/golang/blob/071e264f53e89ea75f1a38f6c1c33641685d8560/1.15/buster/Dockerfile#L18

Reply:
Updated the variables as discussed. Let me know if it makes sense.